spec/components/text_sentinel_spec.rb

```diff
@@ -74,6 +74,11 @@
       expect(TextSentinel.new(valid_string.upcase)).not_to be_valid
     end
 
+    it "allows all caps topics when loud posts are allowed" do
+      SiteSetting.stubs(:allow_uppercase_posts).returns(true)
```

Comment:
Instead of mocking here, could you just say `SiteSetting.allow_uppercase_posts = true`? We reset changes to site settings after every test.